core/system/inc/linker.h

```diff
@@ -19,62 +19,69 @@
 
 #include "api/inc/priv_sys_irq_hook_exports.h"
 
-UVISOR_EXTERN const uint32_t __code_end__;
-UVISOR_EXTERN const uint32_t __stack_start__;
-UVISOR_EXTERN const uint32_t __stack_top__;
-UVISOR_EXTERN const uint32_t __stack_end__;
+extern uint32_t const __code_end__;
+extern uint32_t const __stack_start__;
+extern uint32_t const __stack_top__;
+extern uint32_t const __stack_end__;
 
-UVISOR_EXTERN uint32_t __bss_start__;
-UVISOR_EXTERN uint32_t __bss_end__;
+extern uint32_t __bss_start__;
+extern uint32_t __bss_end__;
 
-UVISOR_EXTERN uint32_t __data_start__;
-UVISOR_EXTERN uint32_t __data_end__;
-UVISOR_EXTERN const uint32_t __data_start_src__;
+extern uint32_t __data_start__;
+extern uint32_t __data_end__;
+extern uint32_t const __data_start_src__;
 
-UVISOR_EXTERN void main_entry(void);
-UVISOR_EXTERN void isr_default_sys_handler(void);
-UVISOR_EXTERN void isr_default_handler(void);
-UVISOR_EXTERN int isr_default(uint32_t isr_id);
+extern void main_entry(void);
+extern void isr_default_sys_handler(void);
+extern void isr_default_handler(void);
+extern int isr_default(uint32_t isr_id);
 
 typedef struct {
     uint32_t magic;
     uint32_t version;
-    uint32_t *mode;
+    uint32_t * mode;
 
-    /* protected bss */
-    uint32_t *bss_start, *bss_end;
+    /* Protected BSS section */
+    uint32_t * bss_start;
+    uint32_t * bss_end;
 
-        /* uvisor main bss */
-    uint32_t *bss_main_start, *bss_main_end;
+        /* uVisor own BSS section */
+    uint32_t * bss_main_start;
+    uint32_t * bss_main_end;
 
-        /* secure boxes bss */
-    uint32_t *bss_boxes_start, *bss_boxes_end;
+        /* Seecure boxes BSS section */
+    uint32_t * bss_boxes_start;
+    uint32_t * bss_boxes_end;
 
-    /* uvisor code and data */
-    uint32_t *main_start, *main_end;
+    /* uVisor own code and data */
+    uint32_t * main_start;
+    uint32_t * main_end;
 
-    /* protected flash memory region */
-    uint32_t *secure_start, *secure_end;
+    /* Protected flash memory region */
+    uint32_t * secure_start;
+    uint32_t * secure_end;
 
-        /* boxes configuration tables */
-    uint32_t *cfgtbl_start, *cfgtbl_end;
+        /* Secure boxes configuration tables */
+    uint32_t * cfgtbl_start;
+    uint32_t * cfgtbl_end;
 
-        /* pointers to boxes configuration tables */
-    uint32_t *cfgtbl_ptr_start, *cfgtbl_ptr_end;
+        /* Pointers to the secure boxes configuration tables */
```

Comment:
Do we want comments not aligned with the code still, or should we fix that in this commit, too?